src/java.base/share/classes/java/io/PrintStream.java

```diff
@@ -45,8 +45,8 @@
  * ({@code '\n'}) is written.
  *
  * <p> All characters printed by a {@code PrintStream} are converted into
- * bytes using the given encoding or charset, or the platform's default
- * character encoding if not specified.
+ * bytes using the given encoding or charset, or the default
+ * console charset if not specified.
```

Comment:
JEP 400 doesn't give a rationale for using the console charset for PrintStream.
PrintStreams are used for output to files and other media other than just a tty/console.
The charset of system.out/err should use the console charset.

Reply:
This was my thinking in https://github.com/openjdk/jdk/pull/4733#issuecomment-876793372.

Reply:
OK, I am now conviced. Modified not to default to Console.charset() for generic PrintStream w/o charset constructor.